client/src/components/schedule/schedule/Schedule.jsx

```diff
@@ -0,0 +1,175 @@
+  const circleRef = useRef();
+
+  const [x, setX] = useState();
+  const [y, setY] = useState();
+  const getPosition = () => {
+    const x = circleRef.current.offsetLeft;
+    const y = circleRef.current.offsetTop;
+    setX(x);
+    setY(y);
+  };
+
+  useEffect(() => {
+    getPosition();
+  });
+
+  const [windowDimensions, setWindowDimensions] = useState(getWindowDimensions());
+  function getWindowDimensions() {
+    const { innerWidth: width, innerHeight: height } = window;
+    return {
+      width,
+      height,
+    };
+  }
+
+  useEffect(() => {
+    function handleResize() {
+      setWindowDimensions(getWindowDimensions());
+    }
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+
+  for (var i = 0; i < scheduleList.length; i++) {
+    let day = scheduleList[i].date.slice(0, scheduleList[i].date.indexOf(' '));
+    let date = scheduleList[i].date.slice(scheduleList[i].date.indexOf(' '));
+    var buttonDayDate;
+    if (windowDimensions.width > 860) {
+      buttonDayDate = (
+        <div className={'schedule-day-date-desktop'}>
+          <div>{day}</div>
+          <div className={'schedule-date'}>{date}</div>
+        </div>
+      );
+    } else {
+      buttonDayDate = (
+        <div className={'schedule-day-date-mobile'}>
+          <div>{day}</div>
+        </div>
+      );
+    }
+    var dateVal = { name: [buttonDayDate] };
+    days.push(dateVal);
+    const dayList = scheduleList[i].events.map((dayData, index) => (
+      <div
+        className={
+          windowDimensions.width > 860 ? 'schedule-events-desktop' : 'schedule-events-mobile'
+        }
+        key={index}
+      >
+        <ScheduleAccordionWrapper
+          scheduleData={dayData}
+          index={index}
+          windowDimensions={windowDimensions}
+        />
+      </div>
+    ));
+    dayEvents.push(dayList);
+  }
+
+  for (var j = 0; j < scheduleList.length; j++) {
+    outerCircleIcons.push(
+      <div
+        className={windowDimensions.width > 860 ? 'schedule-circle-outer' : ''}
+        style={{ top: `${115 * j}px` }}
+      ></div>,
+    );
+    innerCircleIcons.push(
+      <div
+        className={windowDimensions.width > 860 ? 'schedule-circle-inner' : ''}
+        style={{ top: `${115 * j}px` }}
+      ></div>,
+    );
+  }
+
+  console.log(days);
+  return (
+    <div className={'schedule-container'}>
+      <div
+        className={
+          windowDimensions.width > 860
+            ? 'schedule-circle-container-desktop'
+            : 'schedule-circle-container-mobile'
+        }
+      >
+        <div ref={circleRef}>{outerCircleIcons}</div>
+        <svg
+          className={windowDimensions.width > 860 ? 'schedule-circle-line' : 'schedule-hide'}
+          height={115 * (scheduleList.length - 1) + 15 * scheduleList.length}
+          strokeWidth="20"
+        >
+          <line y1={45} y2={1000} stroke="#ffe863" />
```

Comment:
Avoid inline colors
Especially when we have a color dictionary in `App.scss`

Reply:
Replaced stoke with CSS class that defines the stoke with var(--color)